drivers/gles3/rasterizer_canvas_gles3.cpp

```diff
@@ -1500,7 +1500,7 @@ void RasterizerCanvasGLES3::render_joined_item(const BItemJoined &p_bij, RenderI
 					}
 				}
 
-				glBindBufferBase(GL_UNIFORM_BUFFER, 1, static_cast<LightInternal *>(light->light_internal.get_data())->ubo);
+				glBindBufferBase(GL_UNIFORM_BUFFER, 1, static_cast<LightInternal *>(light_internal_owner.get(light->light_internal))->ubo);
 
```

Comment:
This is the other occasion when `get_data()` had to be changed as it is no longer supported directly on the RID as it won't compile with `tracked_handles`. This form compiles down to the same form as previous anyway when `rids=pointers` (i.e. default) is set.